distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/ServiceManagerImpl.java

```diff
@@ -66,10 +68,20 @@
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
+import com.google.common.collect.ImmutableMap;
 import com.google.common.collect.Sets;
 import com.jayway.restassured.response.Response;
 
 public class ServiceManagerImpl implements ServiceManager {
+    private static final Map<Integer, String> BUNDLE_STATES =
+            new ImmutableMap.Builder<Integer, String>().put(Bundle.UNINSTALLED, "UNINSTALLED")
```

Comment:
@shaundmorris You were correct. I'm not sure how I dropped off the UNINSTALLED state when I copied those over. 

I did have to change to using a `Builder` to make the map because the overloaded `of` methods on `ImmutableMap` only support up to five entries.